cordova-lib/spec-plugman/plugins/org.test.defaultvariables/plugin.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<plugin xmlns="http://cordova.apache.org/ns/plugins/1.0"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    id="org.test.defaultvariables"
+    version="3.0.0">
+
+    <name>Use Default Variables</name>
+
+    <!-- android -->
+    <platform name="android">
+            <preference name="ENABLE_LOG" default="yes" />
+            <preference name="API_KEY" />
+    </platform>
```

Comment:
If you want to override the default variable, just specify it during the plugin installation : 
ex :` cordova plugin add <plugin_id> --variable ENABLE_LOG=no`  
This will update the `cordova/plugins/android.json`accordingly, as this is where the variables are being stored (and not in the config.xml)